worker/test/src/RTC/TestRtpPacket.cpp

```diff
@@ -502,6 +503,7 @@ SCENARIO("parse RTP packets", "[parser][rtp]")
 			0x00, 0x00, 0x00, 0x00,
 			0x00, 0x00, 0x00, 0x00,
 			0x00, 0x00, 0x00, 0x00,
+			0x00, 0x00, 0x00, 0x00,
```

Comment:
Might have been beneficial to expand comment on why extra buffer is needed and maybe have buffer size be a fixed size array calculate statically, otherwise this seems to be somewhat accidental that it works at all. In other tests you've set buffer to 1500 bytes, but not here.

Reply:
Totally agree, but I don't know how the buffer is calculated.

Reply:
> In other tests you've set buffer to 1500 bytes, but not here.

Yes, let's be consistent and just defined the buffer length rather than adding this extra zeroes please.